### PR TITLE
Show the document next to the translation

### PR DESCRIPTION
**Pull request by mohamedlafiteh**

### CODE REVIEW

File: client/src/components/AddTranslationForm.js [hunk 1]
```diff
@@ -29,6 +32,7 @@ export class AddTranslationForm extends Component {
     let documentId = this.props.match.params.documentId;
     getDocumentById(documentId)
       .then(document => {
+        console.log(document);
```
Comment (NadaKartouch) [inline client/src/components/AddTranslationForm.js]: can you please remove console log

File: client/src/components/ViewDocument.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ export class ViewDocument extends Component {
     this.state.translation
       ? (translationContent = this.state.translation.content)
       : (translationContent = "");
-
+    //console.log(this.state.document.content);
```
Comment (NadaKartouch) [inline client/src/components/ViewDocument.js]: can u please remove commented out console log

File: client/src/components/AddTranslationForm.js (hunk 1 above)
Comment (mohamedlafiteh) [reply client/src/components/AddTranslationForm.js]: it is done

File: client/src/components/ViewDocument.js (hunk 2 above)
Comment (mohamedlafiteh) [reply client/src/components/ViewDocument.js]: I removed it

**Pull request merged**